native/src/components/EventListItem.tsx

```diff
@@ -25,19 +26,18 @@ const Description = styled.Text`
 const placeholderThumbnails = [EventThumbnailPlaceholder1, EventThumbnailPlaceholder2, EventThumbnailPlaceholder3]
 
 const getDateIcon = (date: DateModel): { icon: JSXElementConstructor<SvgProps>; label: string } | null => {
-  const isRecurring = date.hasMoreRecurrencesThan(1)
-  const isToday = date.isToday
-
-  if (isRecurring && isToday) {
-    return { icon: CalendarTodayRecurringIcon, label: 'todayRecurring' }
-  }
-  if (isRecurring) {
-    return { icon: CalendarRecurringIcon, label: 'recurring' }
-  }
-  if (isToday) {
-    return { icon: CalendarTodayIcon, label: 'today' }
+  const nativeIcons: { [key in EventIcon]: JSXElementConstructor<SvgProps> } = {
```

Comment:
I think the prefix `native` is confusing bc there are no web icons here, just `icons` would be enough imo:
```suggestion
  const icons: { [key in DateIcon]: JSXElementConstructor<SvgProps> } = {
```